charts/hathor-fullnode/values.yaml

```diff
@@ -0,0 +1,103 @@
+# Default values for hathor-full-node.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+hathor:
+  network: testnet
+  snapshotUrl: https://hathor-public-files.s3.amazonaws.com/temp/testnet-data-latest
+  # For mainnet:
+  # snapshotUrl: https://hathor-public-files.s3.amazonaws.com/temp/mainnet-data-latest
+
+  # Check https://github.com/HathorNetwork/hathor-core/blob/master/hathor/cli/run_node.py
+  additionalOptions:
+    cache:
+      enabled: true
+      size: '100000'
+    xFastInitBeta: true
+    walletIndex: true
+
+statefulset:
+  # You should avoid using more than 1 replica. The application is not prepared to handle concurrent access in the database.
+  replicas: 1
+
+image:
+  repository: hathornetwork/hathor-core
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+envFrom: {}
+  # - configMapRef:
+  #     name: config-map-name
+  # - secretRef:
+  #     name: secret-name
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+persistentVolumeClaim:
+  storageClassName: gp2
+  resources:
+    requests:
+      storage: 30Gi
```

Comment:
30GB is what we have nowadays as RDS?

Reply:
This is the storage to be used by the full-node. It will contain the full-node database.

It's not related to RDS.